Library/Formula/wellington.rb

```diff
@@ -17,35 +17,58 @@ class Wellington < Formula
 
   depends_on "go" => :build
   depends_on "pkg-config" => :build
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
 
-  go_resource "github.com/wellington/spritewell" do
-    url "https://github.com/wellington/spritewell.git",
-        :revision => "748bfe956f31c257605c304b41a0525a4487d17d"
+  stable do
+    depends_on "libsass"
   end
 
-  go_resource "github.com/go-fsnotify/fsnotify" do
-    url "https://github.com/go-fsnotify/fsnotify.git",
-        :revision => "f582d920d11386e8ae15227bb5933a8f9b4c3dec"
+  devel do
+    url "https://github.com/wellington/wellington/archive/c574754c39806e8c52682f15f49c8ff819d0f962.tar.gz"
+    sha256 "92c5d4bdd6260f96aed0e20fa06c6200b1c57281dbd32dbd00b1c06636ef6e10"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
```

Comment:
Go build expects quotes around -X main.version. The output of wt -v reports
something not 0.8.1 when I did that.
On Tue, May 5, 2015 at 6:02 PM tim smith <notifications@github.com> wrote:

> In Library/Formula/wellington.rb
> <https://github.com/Homebrew/homebrew/pull/39195#discussion_r29717920>:
>
> >    end
> >
> > -  go_resource "github.com/go-fsnotify/fsnotify" do
> > -    url "https://github.com/go-fsnotify/fsnotify.git",
> > -        :revision => "f582d920d11386e8ae15227bb5933a8f9b4c3dec"
> > +  devel do
> > +    url "https://github.com/wellington/wellington/archive/c574754c39806e8c52682f15f49c8ff819d0f962.tar.gz
>
> "
> > +    sha256 "92c5d4bdd6260f96aed0e20fa06c6200b1c57281dbd32dbd00b1c06636ef6e10"
> > +
> > +    depends_on "autoconf" => :build
> > +    depends_on "automake" => :build
> > +    depends_on "libtool" => :build
>
> you should repeat these dependencies in a head do block
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/39195/files#r29717920>.
>